search/src/Components/dataset/dataset_list.js

```diff
@@ -99,6 +100,7 @@ export default function DatasetList(props) {
                 <TableCell>Pasture</TableCell>
                 <TableCell></TableCell>
                 <TableCell>{row.contributor}</TableCell>
+                {inReview ? <TableCell>{row.contributor_email}</TableCell> : ""}
```

Comment:
Link it to make contact easy?
```suggestion
                {inReview ? <TableCell><a href="mailto:{row.contributor_email}">{row.contributor_email}</a></TableCell> : ""}
```